test/installation/package.json

```diff
@@ -26,7 +26,7 @@
       ]
     },
     "build:packages": {
-      "command": "npm pack --workspace puppeteer --workspace puppeteer-core --workspace @puppeteer/browsers",
+      "command": "npm pack --quiet --workspace puppeteer --workspace puppeteer-core --workspace @puppeteer/browsers",
```

Comment:
Make the `pack` only report the output files and not every file it tries to compress.